package.json

```diff
@@ -112,7 +112,7 @@
     "bash": "0.0.1",
     "bluebird": "^3.3.3",
     "body-parser": "^1.14.1",
-    "capitano": "^1.7.0",
+    "capitano": "^1.8.2",
```

Comment:
Version bumped to take advantage of a bug fix: https://github.com/balena-io/capitano/pull/68